### ISSUE TITLE
compiled version

### ISSUE THREAD

**Issue by steto123** [3dmol/3Dmol.js#294]
Hallo,

i have preoblems to compile this project. Ist a precompiled version available for download?

Many Thanks

**Comment by dkoes** [comment]
This is Javascript.  There is no compiler.  Minified versions are available in the build directory or through the CDN, as described in the documentation.
http://3dmol.csb.pitt.edu/doc/index.html

**Comment by steto123** [comment]
Sorry,
This is my problem, i can not use grunt on windows and grunt build the javascript. We have checked this also on linuxmint and we got many errors.
At this time i use the hosted version https://3Dmol.csb.pitt.edu/build/3Dmol-min.js with my wordpress installation.  I build chemistry related pages since 1995 an i can better sleep, when i have this script on my own site. Thats the reason for my question.
